oracle.go

```diff
@@ -146,6 +153,7 @@ func (o *DBOracle) prepareWorkingDirectly() error {
 		docsToInsert = docsToInsert[:0]
 		return err
 	}
+	count := uint64(0)
```

Comment:
We do not appear to be doing anything with this...

Reply:
you're right. Original count was used to determine shouldRetry, but that was switched in favor of just checking Oracle.Count()